basic-tutorial/7_wrapping-up.md

```diff
@@ -99,4 +99,4 @@ Now that you've learned the basics of PennController, we recommend reading the [
 + Create obligatory checkboxes and global variables.
 + Examine PennController results files.
 
-You can also jump right in and [start creating your own experiments](https://expt.pcibex.net/){:target="_blank"}. If you have questions, visit the [**Troubleshooting**]({{site.baseurl}}/docs/troubleshooting) page, check out our [**How-to guides**]({{site.baseurl}}/docs/how-to-guides), or post a question on our [support forum](https://www.pcibex.net/forums/){:target="_blank"}.
+You can also jump right in and [start creating your own experiments](https://expt.pcibex.net/){:target="_blank"}. If you have questions, visit the [**Troubleshooting**]({{site.baseurl}}/docs/troubleshooting) page, check out our [**How-to guides**]({{site.baseurl}}/docs/how-to-guides), or post a question on our [support forum](https://farm.pcibex.net/experiments/new){:target="_blank"}.
```

Comment:
I don't think you needed to change this link, it's meant to link to the [current support forum](https://www.pcibex.net/forums/).

Reply:
good catch---maybe you could you fix it in your next commit along with the "_newtab" targets?